provider/cmd/pulumi-resource-pulumiservice/schema.json

```diff
@@ -554,6 +554,17 @@
         }
       ]
     },
+    "pulumiservice:index:DeploymentSettingsCacheOptions": {
+      "description": "Dependency cache settings for the deployment",
+      "properties": {
+        "enabled": {
```

Comment:
Thanks for catching this in the docs PR!
Should be `enable`, without the d, to match the API

(I personally also prefer `enabled` but consistency is more important)

Reply:
updated! https://github.com/pulumi/pulumi-pulumiservice/pull/436/commits/d019a21572b11d03d52c751545f386568f9bf07a